src/Whip_MessageDismisser.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+
+/**
+ * A class to dismiss messages
+ */
+final class Whip_MessageDismisser {
```

Comment:
This cannot be a final class without an `abstract` class or `interface` implementation.